tiny_dnn/core/kernels/tiny_quantized_conv2d_kernel.h

```diff
@@ -39,66 +39,28 @@ inline void tiny_quantized_conv2d_kernel(const conv_params& params,
                                          const vec_t&       bias,
                                          vec_t&             a,
                                          const bool layer_parallelize) {
-    // image quantization
-    float_t min_input(in[0]);
-    float_t max_input(in[0]);
-    for (serial_size_t inc = 0; inc < params.in.depth_; inc++) {
-        for (serial_size_t ins = 0; ins < params.in_padded.height_*params.in_padded.height_; ins++) {
-            serial_size_t idx = params.in_padded.get_index(0, 0, inc);
-            min_input = std::min(min_input, (&in[idx])[ins]);
-            max_input = std::max(max_input, (&in[idx])[ins]);
-        }
-    }
+    // Input quantization
+    vec_t range_in = tensor_range<float_t>(in);
```

Comment:
maybe we can go for a inherited `QuantizedTensor` or if there is a real need, just add a small attributes to Tensor regarding data ranges.